content/en/docs/concepts/observability/index.md

```diff
@@ -118,6 +118,78 @@ of Istio itself (as distinct from that of the services within the mesh).
 
 For more information on which metrics are maintained, please refer to the [reference documentation](/docs/reference/commands/pilot-discovery/#metrics).
 
+### Gathering metrics using Prometheus Operator
```

Comment:
This is not the right place for this. This is supposed to be a high level overview of observability. https://preliminary.istio.io/latest/docs/ops/integrations/prometheus seems more fitting if we are going to add this

Reply:
In particular it will need to clarify which of the 2 integration strategies it uses

Reply:
@craigbox I can link the samples instead of adding them inline and add the note about using Prom Operator with Monitors in different namespaces.